### PR TITLE
[11.0][MIG] account_invoice_triple_discount

### PR DESCRIPTION
**Pull request by xavierjimenez**
Migration of account_invoice_triple_discount module to version 11.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I still see only 1 commit, without the commit history.

**Comment by xavierjimenez** [comment]
I think it's correct now (finally).

**Comment by pedrobaeza** [comment]
Yeah, great!

**Comment by misern2** [comment]
@xavierjimenez Travis is red.

File: account_invoice_triple_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+You can also use negative values to charge instead of discount:
+
+Unit price: 600.00 ->
+
+  - Disc. 1 = 50% -> Amount = 300.00
+  - Disc. 2 = -5% -> Amount = 315.00
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/10.0
```
Comment (tarteo) [inline account_invoice_triple_discount/README.rst]: Can you please update the version number to 11.0?

File: account_invoice_triple_discount/views/account_invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    <record id="invoice_supplier_triple_discount_form_view" model="ir.ui.view">
+        <field name="name">
+            account.invoice.supplier.triple.discount.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']//tree//field[@name='discount']"
+                   position="after">
+                <field name="discount2" groups="base.group_no_one"/>
```
Comment (tarteo) [inline account_invoice_triple_discount/views/account_invoice_view.xml:25]: Why is it only for the debug group?

**Review by tarteo — CHANGES_REQUESTED** [review]
Just a question about this module. Is this module a tool to make multiple discounts easier?
Because e.g. price 100 * 0.75 = 25 = 100 * 0.5 * 0.5

File: account_invoice_triple_discount/views/account_invoice_view.xml (hunk 2 above)
Comment (misern2) [reply account_invoice_triple_discount/views/account_invoice_view.xml:25]: It's the same group as the discount field defined in the account_invoicing module.

**Review by tarteo — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Please squash your last 3 commits before merging.

**Comment by xavierjimenez** [comment]
Done!

**Review by misern2 — APPROVED** [review]
LGTM 👍

**Pull request merged**